crates/wasm-encoder/Cargo.toml

```diff
@@ -12,12 +12,14 @@ description = """
 A low-level WebAssembly encoder.
 """
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 leb128 = "0.2.4"
 
 [dev-dependencies]
 anyhow = "1.0.38"
 tempfile = "3.2.0"
 wasmparser = { path = "../wasmparser" }
+
+[features]
+default = []
+component-model = []
```

Comment:
I'd naively expect that this would be compiled in by default, but is there a reason you wanted this gated behind a feature in particular?

Reply:
No reason in particular other than I like to have a non-default feature as a warning that this is in flux and basing anything on top of this is likely to break in the future.

Reply:
Ok in that case I think it's fine to avoid features for now. We already expose work-in-progress things (such as module linking!) without features and we revision breaking changes like that through major versions.

Reply:
I've removed the feature.